samples/quickstart/quickstart.py

```diff
@@ -17,46 +17,44 @@
 
 def main(project_id="your-project-id", snapshot_millis=0):
     # [START bigquerystorage_quickstart]
-    from google.cloud import bigquery_storage_v1
+    from google.cloud.bigquery_storage_v1 import client
```

Comment:
What's the reason to change this import? Can we keep the BigQueryReadClient in the `bigquery_storage_v1` package, assuming we're keeping it?

Reply:
Changed it to import the client from `google.cloud.bigquery.storage`.